src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -327,6 +328,7 @@ ${
         return {
           items: facetValues,
           hasNoResults: results ? results.nbHits === 0 : true,
+          canRefine: Boolean(results && results.nbHits > 0),
```

Comment:
instead it should be facetValues.length > 1 I think

Reply:
I think I'm missing somehting. 
Isn't `canRefine` supposed to be on the item level?
My understanding was: `canRefine` tells you whether you can click (i.e. call refine) on a specific item.

Reply:
it's meant for if you can refine the whole widget, every item can be refined normally